### DISCUSSION TITLE
Synology App hangs

### DISCUSSION THREAD

**Discussion by littleneutrino** [lloesche/valheim-server-docker#6] [Q&A]
I have followed the instructions to the letter, using a Synology DS1019+ Running the Latest official release firmware.
It hangs on the 

 DungeonDB Start 182 | stdout

After that it does another debug step but never continues. I can see it has created the world files, and the start_server.sh file is present as well as the adminlist, bannedlist, and permittedlist, and a prefs file. it just neter continues. even after letting it sit there for 30 minutes.

**Comment by lloesche** [comment]
That sounds correct. `DungeonDB Start` is the last message the server will output after startup. This means the server is ready to accept connections. Once you connect to the server there will be more log messages.

**Reply by littleneutrino** [reply]
So i went into Steam and servers and set it as SynologyIP:2456 and steam says its offline and it didnt work.

**Reply by lloesche** [reply]
If you choose port 2456 you have to enter it as `SynologyIP:2457` in Steam. Don't ask me why. See this section for more details: https://github.com/lloesche/valheim-server-docker#steam-server-favorites--lan-play
